__tests__/app.test.js

```diff
@@ -31,5 +32,38 @@ describe('GET /api', () => {
         })
     
     });
-}
-)
\ No newline at end of file
+})
+
+describe('GET /api/articles', () => {
+    test('resolves with 200 status code and returns correct article data', () => {
+        return request(app).get('/api/articles').expect(200)
+        .then((res) => {
+            const articlesArray = res.body.articles;
+            expect(articlesArray).toHaveLength(13);
+
+            articlesArray.forEach((article) => {
+                expect(typeof article.title).toBe('string');
```

Comment:
it would be easier for you and better to read if you used expect.objectContaining or .toMatchObject here